Python/Github_Issues_PRs_Scrapper/.env.example

```diff
@@ -0,0 +1 @@
+GITHUBTOKEN=duieaX12nui@8jivdrlmkvs01nkjvseior1kvmos12345
```

Comment:
Is this your actual token? If yes, kindly remove it from the commits and rebase.

Reply:
Nope😁, it's not my token

Reply:
Still its not supposed to look like this. See the token in this [PR](https://github.com/HarshCasper/Rotten-Scripts/pull/896)